tests/integration/security/rbac/v2/basic/testdata/istio-extended-rbac-v2-rules.yaml.tmpl

```diff
@@ -0,0 +1,72 @@
+# istio-extended-rbac-v2-rules.yaml to enforce access control for both http and tcp services using Istio RBAC v2 rules
```

Comment:
Same here, the name `extended` is kind of very confusing, especially we already have a `basic` one, not clear what's the difference between them.

Reply:
If there is no suggestion on the naming, I think we can keep it as it is. The test function and template file have a short comment describing the test. I mean, it's not clear what `basic` does either :P